app/assets/stylesheets/puppetclasses.scss

```diff
@@ -2,5 +2,39 @@
   .fields {
     border-top: 1px dotted lightgrey;
     padding-top: 10px;
+    &:nth-child(3) {
+      border-top: none;
+    }
+  }
+}
+
+.lookup-keys-container {
+  h6 {
+    margin-top: 0px;
+  }
+
+  .help-block {
+    margin: 0px;
+  }
+
+  input[type=checkbox]{
+    margin-top: 11px;
+  }
+}
+
+.form-group button[data-original-title="Full screen"]{
+  margin-right: 10px;
+}
+
+.expander {
```

Comment:
this should move to application.scss so it is available for the helper on other pages as well